components/molecules/sidebar/components/sidebar.js

```diff
@@ -3,43 +3,87 @@
  */
 
 // React
-import { array, string } from 'prop-types'
+import React, { useState } from 'react'
+import { array, string, object } from 'prop-types'
 
 // UI
 import { Image, Icon, Link, List } from '../../../'
 import { Divider } from '../../../atoms/divider/divider'
 import { ListItem } from '../../../atoms/list/components/listItem'
+import { Avatar } from 'components'
```

Comment:
@esuarezz Using the aliases in components may break the build? These can be added to the Import that imports Image etc ...